app/kumactl/cmd/uninstall/uninstall_transparent_proxy.go

```diff
@@ -48,11 +48,7 @@ func newUninstallTransparentProxy() *cobra.Command {
 				Stdout:        cmd.OutOrStdout(),
 			}
 
-			tp := transparentproxy.V1()
-
-			if args.EbpfEnabled {
-				tp = transparentproxy.V2()
-			}
+			tp := transparentproxy.V2()
```

Comment:
Nit: The same suggestion as earlier. I would prefer to change the name from `V2` as there is no `V1` anymore.